### ISSUE TITLE
Qualitative Data Request

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#48161]
### Data Type Requested

User Surveys

### Date Range Start

01-01-2022

### Date Range End

10-01-2022

### Product Name(s)

Accessibility

### Product Description

The accessibility team's goal is to establish a process to collect a11y feedback from the public. In order to accomplish this goal, we need to...

1. determine whether disabled Veterans are providing feedback through existing feedback mechanisms (and if not, why not?), and 
2. gain insight into disabled Veteran's behaviors, feelings, perspectives, and expectations on providing a11y feedback

We're looking for an export from the Medallia surveys that contain the following keywords:
accessibility, screen reader, screenreader, jaws, nvda, voiceover, assistive technology, accessible, accommodation, 508, disability, magnification, zoom, disabilities, caption, keyboard, braille, siri

@artsymartha68 for awareness! ✨ 

### Product URL(s)

All pages

### VA Email Address(es)

martha.wilkes@va.gov, josh.kim@adhocteam.us

**Comment by ianMcCullough-ob** [comment]
Thanks for submitting this, @joshkimux!

Are there any common misspellings of the above keywords you would like me to include? The search mechanism within Medallia is pretty literal, so if there are common mispellings or alternate usages feel free to include those also.

Lastly, do you have a preference on how the export(s) are delivered? Meaning do you want one master export (will be a .csv) with all terms, or do you want one file per term?

**Comment by joshkimux** [comment]
One export as a .csv would be perfect! And I'm unsure of any common misspellings, curious just to see if this will present any results!

**Comment by ianMcCullough-ob** [comment]
Good morning @joshkimux 

I wrapped up the exports for these terms, I have two of them. One is a .CSV that has all the terms on one tab. The other is an excel doc that has 3 tabs: one with all the above terms provided, one with all those terms WITHOUT "disability" and one with ONLY the term "disability."

I did this because "disability" produced the most records by far; over 3000, compared to the rest which produced less than 100 combined. I wasn't sure if it would be easier to consume the data broken up into these chunks instead of trying to find those other terms in the midst of 3k records on disability. Hope this makes sense.

Lastly, I was unable to share the exports with you, only @artsymartha68. Some of these records contain PII (phone numbers, email addresses, etc) and I can only share that data on the VA network. If you have a VA email address that you use to login to citrix or a GFE, I can share it with that email address.
